velodrome/transform/influx.go

```diff
@@ -56,6 +65,11 @@ func (config *InfluxConfig) CreateDatabase(tags map[string]string) (*InfluxDB, e
 		return nil, err
 	}
 
+	err = dropSeries(client, measurement, config.DB, tags)
```

Comment:
Depends the scope of "no data". Each plugin removes its data for its repository. If you restart one transform, you just lose the data for one repo within (usually) one graph.